### PR TITLE
Update HISTORY.md for branch cut for  8.4.fb

### PR DESCRIPTION
**Pull request by akankshamahajan15**

### CODE REVIEW

**Review by cbi42 — APPROVED** [review]

File: HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+* Start logging non-zero user-defined timestamp sizes in WAL to signal user key format in subsequent records and use it during recovery. This change will break recovery from WAL files written by early versions that contain user-defined timestamps. The workaround is to ensure there are no WAL files to recover (i.e. by flushing before close) before upgrade.
+* Added new property "rocksdb.obsolete-sst-files-size-property" that reports the size of SST files that have become obsolete but have not yet been deleted or scheduled for deletion
+* *Start to record the value of the flag `AdvancedColumnFamilyOptions.persist_user_defined_timestamps` in the Manifest and table properties for a SST file when it is created. And use the recorded flag when creating a table reader for the SST file. This flag is only explicitly record if it's false.
```
Comment (cbi42) [inline HISTORY.md:9]: extra * here

File: HISTORY.md [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+* RocksDB will try to drop range tombstones during non-bottommost compaction when it is safe to do so. (#11459)
+* When a DB is openend with `allow_ingest_behind=true` (currently only Universal compaction is supported), files in the last level, i.e. the ingested files,  will not be included in any compaction. (#11489)
+* Statistics `rocksdb.sst.read.micros` scope is expanded to all SST reads except for file ingestion and column family import (some compaction reads were previously excluded).
+
+### Bug Fixes
+* Reduced cases of illegally using Env::Default() during static destruction by never destroying the internal PosixEnv itself (except for builds checking for memory leaks). (#11538)
+* Fix extra prefetching during seek in async_io when BlockBasedTableOptions.num_file_reads_for_auto_readahead_ = 1 leading to extra reads than required.
+* Fix a bug where compactions that are qualified to be run as 2 subcompactions were only run as one subcompaction. 
+* *Fix a use-after-move bug in block.cc.
```
Comment (cbi42) [inline HISTORY.md:34]: same here

File: HISTORY.md (hunk 1 above)
Comment (jowlyzhang) [inline HISTORY.md:9]: I incorrectly added a few duplicated stars, would you mind helping remove this and there are two more below. Thanks a lot!

**Review by jowlyzhang — APPROVED** [review]

File: HISTORY.md (hunk 1 above)
Comment (akankshamahajan15) [reply HISTORY.md:9]: Damn! I trusted the script and forgot to check the notes manually.

**Pull request closed without merging**